src/app/pages/user-profile/user-profile.component.html

```diff
@@ -78,7 +78,8 @@ <h3>
               <h3 class="mb-0">My account</h3>
             </div>
             <div class="col-4 text-right">
-              <a href="#!" class="btn btn-sm btn-primary">Settings</a>
+              <a class="col" [routerLink]="['/setting']" class="btn btn-sm btn-primary">Settings</a>
```

Comment:
User profie shall not be the place to add the link. It shall be from the dropdowns.